### PR TITLE
(DOCS) Update to sync with DITA source

### PR DESCRIPTION
**Pull request by katelopresti**
Also removed Release Notes files. This information is not tracked in GitHub.

### CODE REVIEW

File: pre-docs/bolt_configure_orchestrator.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ mod 'puppetlabs-bolt_shim', '0.1.1'
 ## Assign PE RBAC permissions for all tasks
 
 Warning: By granting users access to Bolt tasks, you give them permission to
-run arbitrary commands and upload files as a super-user.  In the PE console,
+run arbitrary commands and upload files as a super-user.
+
+1.  In the PE console,
 click Access control > User roles.
```
Comment (adreyer) [inline pre-docs/bolt_configure_orchestrator.md]: There shouldn't be a newline here

File: pre-docs/inventory_file.md [hunk 2]
```diff
@@ -174,6 +174,7 @@ Configure login and escalation for a specific node.
 nodes:
   - name: host1.example.com
     config:
+      transports:
```
Comment (MikaelSmith) [inline pre-docs/inventory_file.md]: This actually shouldn't be added, what's here already is correct.

File: pre-docs/bolt_options.md [hunk 3]
```diff
@@ -203,10 +203,12 @@ Options are optional unless marked as required.
...
-| `--query, -q` | Query PuppetDB to determine the targets. |
-| `--noop` | Execute a task that supports it in noop mode. |
-| `--description` | Add a description to the run. Used in logging and submitted to Orchestrator with the pcp transport. |
-| `--params` | Parameters, passed as a JSON object on the command line, or as a JSON parameter file, prefaced with @ like @params.json |
+| `--query` `, -q` |Query PuppetDB to determine the targets.|
+| `--noop` |Execute a task that supports it in no-operation mode.|
+| `--description` |Add a description to the run. Used in logging and submitted to Orchestrator with the pcp transport.|
+| `--params` |Parameters, passed as a JSON object on the command line, or as a JSON parameter file, prefaced with `@` like `@params.json`. For Windows PowerShell,  add single quotation marks to define the file: `'@params.json'`
```
Comment (MikaelSmith) [inline pre-docs/bolt_options.md]: Extra space present before `add single quotation`.

File: pre-docs/plan_logging.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Plan Logging
```
Comment (MikaelSmith) [inline pre-docs/plan_logging.md]: Was this moved out of `writing_plans.md`?

**Comment by MikaelSmith** [comment]
@katelopresti we should pair on improving this.

**Pull request merged**